### ISSUE TITLE
Review annotations using extension 'happens_during' but NOT a biological phase

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-annotation#2585]
According to the current version of the GO-CAM specifications, the extension 'happens_during' should only be used with biological phases or life stages. 

Annotations using GO terms other than children of biological phases should be reviewed: 
https://docs.google.com/spreadsheets/d/1qoxnXM3q59jcZLIxnX5SY0H5syfK7GJc38s0VGxx2jM/edit#gid=0

Suggestions: 
- Change the relation 'happens_during'  for GO:P that are not a biological phase to 'part of' or 'causally upstream of or within' (or remove); 
- review and remove UBERON if NOT a life cycle stage

Group | Annotations to review
-- | --
AgBase | 36
ARUK-UCL | 24
BHF-UCL | 1
CAFA | 26
dictyBase | 20
GO_Central | 1
MGI | 1
ParkinsonsUK-UCL | 19
PomBase | 132
SGD | 56
UniProt | 110
WB | 7

Thanks, Pascale

**Comment by ValWood** [comment]
Is the spreadsheet correct?

Most of ours appear to be phases at first glance


protein | taxon:4896 | 20180122 | PomBase | happens_during(GO:0000087 mitotic M phase),has_direct_input(PomBase:SPAC1F5.04c),part_of(GO:1903500) |   |   |   |   |  
-- | -- | -- | -- | -- | -- | -- | -- | -- | --
protein | taxon:4896 | 20170706 | PomBase | happens_during(GO:0000087 mitotic M phase),has_direct_input(PomBase:SPAC20G8.05c) |   |   |   |   |  
protein | taxon:4896 | 20170706 | PomBase | happens_during(GO:0000087 mitotic M phase),has_direct_input(PomBase:SPAC222.10c) |   |   |   |   |  
protein | taxon:4896 | 20170706 | PomBase | happens_during(GO:0000087 mitotic M phase),has_direct_input(PomBase:SPAC22E12.19) |   |   |   |   |  
protein | taxon:4896 | 20170706 | PomBase | happens_during(GO:0000087 mitotic M phase),has_direct_input(PomBase:SPAC22H10.11c) |   |   |   |   |  
protein | taxon:4896 | 20170706 | PomBase | happens_during(GO:0000087 mitotic M phase),has_direct_input(PomBase:SPAC23A1.06c) |   |   |   |   |

**Comment by pgaudet** [comment]
@ValWood 
I put on comment in the left-most column: Either "Not a biological phase - review", or "OK". 

Under the 'TO REVIEW', you have for example: 

PomBase | SPBC19C2.09 | sre1 |   | GO:0001227 | PMID:16537923 | IMP |   | F | sterol regulatory element binding protein, transcription factor Sre1 |   | protein | taxon:4896 | 20111221 | PomBase | happens_during(GO:0071456),has_regulation_target(PomBase:SPCC4G3.17) |  
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
PomBase | SPBC211.04c | mcm6 |   | GO:0005515 | PMID:12006645 | IPI | PomBase:SPAC24H6.06 | F | MCM complex subunit Mcm6 | mis5 | protein | taxon:4896 | 20150701 | PomBase | happens_during(GO:0007089 traversing start control point of mitotic cell cycle) |  
PomBase | SPBC216.07c | tor2 |   | GO:0004674 | PMID:22976295 | IDA |   | F | serine/threonine protein kinase Tor2 | SPBC646.01c | protein | taxon:4896 | 20151027 | PomBase | happens_during(GO:0071242),has_direct_input(PomBase:SPCC4G3.08) |  
PomBase | SPBC27B12.11c | pho7 |   | GO:0001228 | PMID:28811350 | IMP |   | F | transcription factor Pho7 |   | protein | taxon:4896 | 20180817 | PomBase | happens_during(GO:0016036) |  

(Note that I wasn't able to put in all GO terms for the extensions; I only put the terms for phases, and annotations that don't have phases are flagged). 

Pascale

**Comment by ValWood** [comment]
Still quite a lot of these are phases, even in the review set.

I still see quite a few annotations that are phases in the "review" list

happens_during(GO:0070314 G1 to G0  transition),has_direct_input(PomBase:SPAC3G9.09c)
happens_during(GO:0070314   G1 to G0 transition)

happens_during(GO:0098764 meiosis I cell cycle phase),has_direct_input(PomBase:SPAC22F3.03c),part_of(GO:0010845)

happens_during(GO:0072690, single celled   vegetative growth phase)

Also, although many examples provide the GO term, many only have the ID and this makes it difficult to check because each one needs to be looked at independent. Could you gnerate a version of the spreadsheet  where the term name is always included?

**Comment by ValWood** [comment]
We  use happens_during with phases, but also with  happens_during "response to specific stresses". This  allows us to differentiate between events that are _involved in_ a stress response and events which occur at the same time as a stress response but we do not know if they are part of the stress response  or just happening simultaneously.

e.g.

![during](https://user-images.githubusercontent.com/7359272/63028241-5ae02b80-bea6-11e9-84cd-8bcb425a21c5.jpg)

that said, I think quite a few of the annotations using happens_during stress with a signalling pathway could probably be converted to part_of. 

One reason I didn't like to do this is that a single pathway can respond to multiple stresses, its just the same pathway being activated in different conditions. If you fully precompose the part_of stress extension (which is semantically equivalent), you and up with many independent terms to describe the same process/pathway. So happens during seems neater, since it is (often) only one of many conditions which can activate the pathway

**Comment by ValWood** [comment]
So  in the above example the precomposed term  would include the target gene. We do not really know that this is involved  in the cellular response to cadmium ion, but we know it is expressed during  cadmium stress.

**Comment by pgaudet** [comment]
> We use happens_during with phases, but also with happens_during "response to specific stresses". This allows us to differentiate between events that are involved in a stress response and events which occur at the same time as a stress response but we do not know if they are part of the stress response or just happening simultaneously.


This is how we thought 'causally upstream of or within should be used.

**Comment by ValWood** [comment]
But "causally upstream of or within "
wouldn't make sense in an extension would it?  I don't understand?
These are for connections of an MF to a  "response"

I really believe "responses  to" terms should be treated like phases, because they aren't really processes, they are collections of processes.  The cause many problems for annotation when used directly.

**Comment by pgaudet** [comment]
> I really believe "responses to" terms should be treated like phases, because they aren't really processes, they are collections of processes. 

So, we would not allow direct annotation ? We should discuss this  on a future call. We anyway wanted to tackle response to at some point - that may be an opportunity.

**Comment by ValWood** [comment]
I still don't really understand "causally upstream of or within"
I don't know the boundaries (also this isn't really necessarily 'causally upstream or within'
it is usually downstream. It's just "happening at the same time" as far as we know.

We have so many problems with response to terms, I would prefer not to muddy the water by making these "involved in". I was hoping we were going to  deal with these differently later- but this discussion goes back a few years.

I don't want to end up in the mouse/human situation where over half the genome are annotated to "response to" terms . We would not want to make all of these annotations  direct to "response_to" terms using causally upstream (even if it was correct to do so, and I dont think it is).

I liked @cmungall m suggestion that these move eventually to an  exposure/event ontology for the 
response to chemical
https://github.com/geneontology/go-ontology/issues/14303

One option is for PomBase to filter all happens_during from the GO submission.
I have fixed, or requested a fix for those which are not phases or responses.

Responses are really tricky because they are defined so vaguely in GO as a 'change'. this means they can be part_of, involved_in or acts_downstrream_of

I particularly dislike the  (potential) instantiation of many variants of the same pathway that would occur by pre-composition of these if they were used as part_of extensions.

**Comment by ValWood** [comment]
As a stop-gap can we allow "response to x" terms in extensions with happens during until the fate of the response to terms is decided. Otherwise I sense a duplication of work. Many times we move annotations to one place, and then need to move them somewhere else later.

**Comment by pgaudet** [comment]
As far as I can tell, there are 120 annotations or so by pombase (not sure they are all 'response to'). You can leave them for now until we discuss it further.

**Comment by ValWood** [comment]
I have been through them and deleted the ones which are not  "phase" or "response to"
Note that many which required checking were actually phases (these terms are listed above)

**Comment by pgaudet** [comment]
@vanaukenk and I are discussing this, and until we have a solution we will allow 'response_to' terms with 'happens_during'. 

Thanks, 
Pascale

**Comment by pfey03** [comment]
done with dicty

**Comment by RLovering** [comment]
I think we need a more specific definition of a 'phase' as many 'processes' are just collections of 'smaller processes'. Once response to becomes a phase then all development terms needs to be considered as phases. Which might be fine, but it depends what restrictions then get imposed and why we need to define something as a phase rather than a process.

**Comment by pfey03** [comment]
@RLovering Yes, while I changed all of ours and it was kind of possible, I agree. In Dicty, I consider developmental stages also phases. It's hard to define what's a stage, a phase, or any developmental process.

**Comment by ValWood** [comment]
My interpretation
 (@cmungall  will be better to comment on this).

This is a general ontology issue and has only been resolved by GO for the terms under "phase"

GO:0044848    biological phase
A distinct period or stage in a biological process or cycle.

which is defined temporally, not as a process.

These term have a different meaning from processes,  although for historical and pragmatic reasons it is housed in the BP ontology.

Developmental terms  in GO are defined  as processes.

Ideally, there would be specialist ontologies  for stages of developmental processes and lifecycle stages. In the absence of these (or their use),  if you use the relation  occurs_during GO:development_term you are slightly changing the meaning of the term to refer to the "developmental stage" rather than the process (I guess they are  interchangable for practical purposes but the meaning is subtly different).

This seems like a pragmatic solution  in the short or medium term.

Note that "I think we need a more specific definition of a 'phase' as many 'processes' are just collections of 'smaller processes'." isn't true.

Phases are not definesd as "collections of processes" (these would be processes). They are defined as specific periods of time (often defined by when specified processes occur). They are not themselves processes. So all of the things in this branch

https://www.ebi.ac.uk/QuickGO/term/GO:0044848
refer to phases
S-phase (the period when DNA replication occurs, but not te process DNA replication)
M-phase (the period when mitosis occurs, but not the actual process of mitosis)
M-phase has sub-phases like GO:0000090    mitotic anaphase (the period when chromosome segregation occurs).

**Comment by RLovering** [comment]
Hi Val
my comment about needing a definition of 'phase' was in response to your statement above: "I really believe "responses to" terms should be treated like phases, because they aren't really processes, they are collections of processes. The cause many problems for annotation when used directly". But I obviously over interpreted your point

**Comment by ValWood** [comment]
Yes, I was unclear.

"Response to" annotations are collections of processes rather than a single process.
For example, response to oxidative stress can encompass

transcription in response to oxidative stress
translation in response to oxidative stress
transport in response to oxidative stress
signaling in response to oxidative stress
etc

When we talk about "response to " in GO we are not  even specific about meaning a  process which is part of the mechanism of the response (as for oxidative stress example above),  many "response to" annotations are only recording a change, without  this change being specific to the response mechanism (it could just be some pathological effect to some toxicity).

Basically, as they stand at the moment they aren't very useful as GO annotations...because they cover both situtation.

**Comment by BarbaraCzub** [comment]
ARUK-UCL and BHF-UCL are done.
ParkinsonsUK-UCL are not done yet.
cc @RLovering

**Comment by BarbaraCzub** [comment]
ParkinsonsUK-UCL have now been reviewed.
cc @RLovering

**Comment by vanaukenk** [comment]
@ValWood  @RLovering @BarbaraCzub - can you remove your github handle from the assignee list if you are all done?  Thanks!

**Comment by BarbaraCzub** [comment]
I removed myself @RLovering.

**Comment by ValWood** [comment]
we have a few 100 of these so this one will take a while. We have a ticket on our tracker.

**Comment by srengel** [comment]
SGD done.  i left as happens_during the ones that used these items in col16:
meiosis I
meiosis II
G2/M transition
meiotic prophase I

**Comment by pgarmiri** [comment]
Hi, the UniProt annotations have been revised.  
Thanks,
Penelope
